web/components/bet-panel.tsx

```diff
@@ -281,92 +304,141 @@ function BuyPanel(props: {
 
   return (
     <Col className={hidden ? 'hidden' : ''}>
-      <div className="my-3 text-left text-sm text-gray-500">
-        {isPseudoNumeric ? 'Direction' : 'Outcome'}
-      </div>
-      <YesNoSelector
-        className="mb-4"
-        btnClassName="flex-1"
-        selected={outcome}
-        onSelect={(choice) => onBetChoice(choice)}
-        isPseudoNumeric={isPseudoNumeric}
-      />
-
-      <Row className="my-3 justify-between text-left text-sm text-gray-500">
-        Amount
-        <span className={'xl:hidden'}>
-          Balance: {formatMoney(user?.balance ?? 0)}
-        </span>
-      </Row>
-
-      <BuyAmountInput
-        inputClassName="w-full max-w-none"
-        amount={betAmount}
-        onChange={onBetChange}
-        error={error}
-        setError={setError}
-        disabled={isSubmitting}
-        inputRef={inputRef}
-        showSliderOnMobile
-      />
+      {!mobileView && (
+        <>
+          <YesNoSelector
+            className="mb-4"
+            btnClassName="flex-1"
+            selected={outcome}
+            onSelect={(choice) => onBetChoice(choice)}
+            isPseudoNumeric={isPseudoNumeric}
+          />
+        </>
+      )}
+      {mobileView && (
+        <>
+          <YesNoSelector
+            className="mb-4"
+            btnClassName="flex-1"
+            selected={outcome}
+            onSelect={(choice) => mobileOnBetChoice(choice)}
+            isPseudoNumeric={isPseudoNumeric}
+          />
+        </>
+      )}
```

Comment:
Seems like these could be combined? You just need to change the `onSelect` prop based on `mobileView`